src/plugins/ui/find_and_replace.ts

```diff
@@ -188,10 +188,9 @@ export class FindAndReplacePlugin extends UIPlugin {
     }
     //modulo of negative value to be able to cycle in both directions with previous and next
     nextIndex = ((nextIndex % matches.length) + matches.length) % matches.length;
-    if (this.selectedMatchIndex === null || this.selectedMatchIndex !== nextIndex) {
-      this.selectedMatchIndex = nextIndex;
-      this.selection.selectCell(matches[nextIndex].col, matches[nextIndex].row);
-    }
+    const nextMatchPosition = { col: matches[nextIndex].col, row: matches[nextIndex].row };
+    this.selectedMatchIndex = nextIndex;
+    this.selection.selectCell(nextMatchPosition.col, nextMatchPosition.row);
```

Comment:
nitpick:  don't know if line 191 is really necessary ? we could have
``` javascript
this.selection.selectCell(matches[nextIndex].col, matches[nextIndex].row);
```
or even 
``` javascript
this.selection.selectCell(matches[this.selectedMatchIndex].col, matches[this.selectedMatchIndex].row);
```

Reply:
Yes.